config/_default/menus/main.en.yaml

```
@@ -4627,386 +4627,456 @@ menu:
     - name: HTTP Server
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/http_server/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_http_server
-      weight: 2015
+      weight: 2017
+    - name: Kafka
+      url: observability_pipelines/set_up_pipelines/dual_ship_logs/kafka/
+      parent: observability_pipelines_dual_ship_logs
+      identifier: observability_pipelines_dual_ship_logs_kafka
+      weight: 2018
     - name: Logstash
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/logstash/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_logstash
-      weight: 2016
+      weight: 2019
     - name: Splunk HTTP Event Collector
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/splunk_hec/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_splunk_hec
-      weight: 2017
+      weight: 2020
     - name: Splunk Forwarders (TCP)
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/splunk_tcp/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_splunk_tcp
-      weight: 2018
+      weight: 2021
     - name: Sumo Logic Hosted Collector
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/sumo_logic_hosted_collector/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_sumo_logic_hosted_collector
-      weight: 2019
+      weight: 2022
     - name: Syslog
       url: observability_pipelines/set_up_pipelines/dual_ship_logs/syslog/
       parent: observability_pipelines_dual_ship_logs
       identifier: observability_pipelines_dual_ship_logs_syslog
-      weight: 2020
+      weight: 2021
     - name: Archive Logs
       url: observability_pipelines/set_up_pipelines/archive_logs/
       parent: observability_pipelines_set_up_pipelines
       identifier: observability_pipelines_archive_logs
       weight: 103
+    - name: Amazon S3
+      url: observability_pipelines/set_up_pipelines/archive_logs/amazon_s3/
+      parent: observability_pipelines_archive_logs
+      identifier: observability_pipelines_archive_logs_amazon_s3
+      weight: 3012
     - name: Datadog Agent
       url: observability_pipelines/set_up_pipelines/archive_logs/datadog_agent/
       parent: observability_pipelines_archive_logs
       identifier: observability_pipelines_archive_logs_datadog_agent
-      weight: 3011
+      weight: 3013
     - name: Fluent
       url: observability_pipelines/set_up_pipelines/archive_logs/fluent/
       parent: observability_pipelines_archive_logs
       identifier: observability_pipelines_archive_logs_fluent
-      weight: 3012
+      weight: 3014
     - name: Google Pub/Sub
       url: observability_pipelines/set_up_pipelines/archive_logs/google_pub_sub/
```

```suggestion
      url: observability_pipelines/set_up_pipelines/archive_logs/google_pubsub/
```
Just happened to catch that this wasn't working in the previews